Source/EnhancedBeliefs/HarmonyPatches.cs

```diff
@@ -109,7 +109,7 @@ public static void Postfix(Pawn_IdeoTracker __instance)
         {
             Pawn pawn = __instance.pawn;
 
-            if (!pawn.Destroyed && pawn.Map != null && __instance.ideo != null && !Find.IdeoManager.classicMode && pawn.IsHashIntervalTick(GenTicks.TickLongInterval))
```

Comment:
NOTE: It's perfectly valid for a pawn to have a null map - if they're being carried, or if they're on a caravan. Are you sure you don't want to calculate these sorts of things if the pawn is unspawned?

Reply:
Oh wow, yeah, I imagine that wouldn't do performance any good!